networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_routing_driver.py

```diff
@@ -588,6 +595,19 @@ def _get_snat_pool_name(self, subnet):
         snat_prefix = self._snat_prefix(subnet)
         return "%s_nat_pool" % (snat_prefix)
 
+    def _add_default_route(self, ri, ext_gw_port):
+        if self._fullsync and (ri.router_id in
+                               self._existing_cfg_dict['routes']):
+            LOG.debug("Default route already exists, skipping")
+            return
+        ext_gw_ip = ext_gw_port['hosting_info']['gateway_ip']
+        if ext_gw_ip:
+            vrf_name = self._get_vrf_name(ri)
+            out_itfc = self._get_interface_name_from_hosting_port(ext_gw_port)
+            conf_str = asr1k_snippets.SET_DEFAULT_ROUTE_WITH_INTF % (
+                vrf_name, out_itfc, ext_gw_ip)
+            self._edit_running_config(conf_str, 'SET_DEFAULT_ROUTE_WITH_INTF')
```

Comment:
Removing the leading underscore doesn't break anything. It seems like this code is never getting called, but I'll keep it like this for now (i.e. remove leading underscore), and we can back it out entirely if/when we it's also removed from the base ASR1k driver.